src/client/components/form-field/form-field.component.js

```diff
@@ -1,54 +1,57 @@
 /* eslint-disable no-console */
 import React, { useState } from 'react';
+import DropDown from '../drop-down/drop-down.component';
 import InputText from '../input-field/input-text.component';
 import InputDate from '../input-field/input-date.component';
 import Button from '../button/button.component';
 
 import './form-field.css';
 
 export default function FormField() {
-  const [plantVariety, setPlantVariety] = useState(null);
-  const [seedPot, setSeedPot] = useState(null);
+  const [cropName, setCropName] = useState(null);
```

Comment:
Consider using an empty string to clear the component because we have a warning in DevTools console: Warning: `value` prop on `select` should not be null.
const [cropName, setCropName] = useState('');

Reply:
Done